Code/EditorPlugins/Assets/EditorPluginAssets/StateMachineAsset/StateMachineGraph.cpp

```diff
@@ -0,0 +1,227 @@
+#include <EditorPluginAssets/EditorPluginAssetsPCH.h>
+
+#include <EditorPluginAssets/StateMachineAsset/StateMachineGraph.h>
+#include <EditorPluginAssets/StateMachineAsset/StateMachineGraphQt.moc.h>
+#include <GameEngine/StateMachine/StateMachine.h>
+#include <GuiFoundation/UIServices/DynamicStringEnum.h>
+
+// clang-format off
+EZ_BEGIN_SUBSYSTEM_DECLARATION(EditorFramework, StateMachine)
```

Comment:
this isn't in the EditorFramework, so the group is wrong, and the dependency on the AssetPlugin is probably also wrong

Reply:
Copy & paste from visual script asset ;)